### PR TITLE
CNV-4300: two RNs and a known issue

### PR DESCRIPTION
**Pull request by ousleyp**
Jira: https://issues.redhat.com/browse/CNV-4300
BZ (this satisfies only a portion of the BZ): https://bugzilla.redhat.com/show_bug.cgi?id=1822266

Preview build: http://file.bos.redhat.com/pousley/042420/cnv-4300-RNs/cnv/cnv_release_notes/cnv-release-notes.html#notable-technical-changes
See the first two "notable technical changes" and the first "known issue"

enterprise-4.4 only

### CODE REVIEW

File: cnv/cnv_release_notes/cnv-release-notes.adoc [hunk 1]
```diff
@@ -85,6 +92,13 @@ xref:../../cnv/cnv_logging_events_monitoring/cnv-events.adoc#cnv-viewing-vm-even
 
 == Known issues
 
+* If you renamed the custom resource when you deployed an earlier version of
+{CNVProductName}, you cannot upgrade directly to {CNVProductName} {CNVVersion}.
+The custom resource must be named `kubevirt-hyperconverged`, which is the default
+name. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822266[*BZ#1822266*])
+** As a workaround, delete the existing custom resource and create a new one.
```
Comment (tiraboschi) [inline cnv/cnv_release_notes/cnv-release-notes.adoc:99]: Sorry, I'd suggest to rename the existing one to the new name (this is allowed!) or create a new one with the right name and only after that delete the original CR.

**Comment by ousleyp** [comment]
attn @nellyc I tagged Tareq in the Jira, but in case Tareq is unavailable, this needs review - thanks!

**Comment by tareqalayan** [comment]
Looks good to me

**Review by aburdenthehand — APPROVED** [review]

**Pull request merged**